.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+  // Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+  // List of extensions which should be recommended for users of this workspace.
```

Comment:
Comments in JSON is discouraged. I think we'd be fine to remove the comments altogether.

Reply:
I'm not against removing the comments but I want to point out...
- The comments are a part of the default file generated by VSCode.
- The functionality works with the comments.
- All the other Sky repos I've already merged this file to has the comments.


### Default File Generation

Default file by using VSCode's command palette:
1. Open project/folder in VSCode.
2. Select `Extensions: Configure Recommended Extensions (Workspace Folder)`
3. A `.vscode/extensions.json` file is generated with the following content.
```
{
	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp

	// List of extensions which should be recommended for users of this workspace.
	"recommendations": [
		
	],
	// List of extensions recommended by VS Code that should not be recommended for users of this workspace.
	"unwantedRecommendations": [
		
	]
}
```